src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/AmqpExtIntegrationTest.php

```diff
@@ -206,4 +221,29 @@ private function createSerializer(): SerializerInterface
             new SerializerComponent\Serializer([new ObjectNormalizer(), new ArrayDenormalizer()], ['json' => new JsonEncoder()])
         );
     }
+
+    private function assertApproximateDuration($startTime, int $expectedDuration)
+    {
+        $actualDuration = microtime(true) - $startTime;
+
+        if (\method_exists([$this, 'assertEqualsWithDelta'])) {
+            $this->assertEqualsWithDelta($expectedDuration, $actualDuration, 'Duration was not within expected range', .5);
```

Comment:
the order of arguments is wrong. `assertEqualsWithDelta($expected, $actual, float $delta, string $message = '')`

Reply:
I'm on it